examples/window-app/common/window-app.zap

```diff
@@ -90,7 +90,7 @@
           "commands": [],
           "attributes": [
             {
-              "name": "IdentifyTime",
+              "name": "identify time",
```

Comment:
Why changing this 
this is already a nightmare to not have everything CamelCased properly in theses files

Reply:
It wasn't my intention. This is what ZAP Tool does when you open and save the file.

Reply:
ok got it !